skrub/tests/test_joiner.py

```diff
@@ -76,25 +71,25 @@ def test_multiple_keys(df_module):
     )
     joiner_list = Joiner(
         aux_table=df2,
-        aux_key=["CO", "CA"],
         main_key=["Co", "Ca"],
+        aux_key=["CO", "CA"],
         add_match_info=False,
     )
     result = joiner_list.fit_transform(df)
 
     expected = ns.concat_horizontal(df, df2)
-    assert_frame_equal(result, expected)
+    df_module.assert_frame_equal(result, expected)
 
     joiner_list = Joiner(
-        aux_table=df2, aux_key="CA", main_key="Ca", add_match_info=False
+        aux_table=df2, main_key="Ca", aux_key="CA", add_match_info=False
     )
     result = joiner_list.fit_transform(df)
-    assert_frame_equal(result, expected)
+    df_module.assert_frame_equal(result, expected)
 
 
-def test_pandas_aux_table_index():
-    main_table = pd.DataFrame({"Country": ["France", "Italia", "Georgia"]})
-    aux_table = pd.DataFrame(
+def test_pandas_aux_table_index(df_module):
```

Comment:
this test is about handling of pandas indexes, so it is really specific to pandas -- there is no need to run it on polars so I would not use df_module here

Reply:
Thanks

Removing the dispatching too to make it clearer